src/externals/global.rs

```diff
@@ -0,0 +1,76 @@
+use crate::{
+    errors::to_py_err,
+    store::Store,
+    types::GlobalType,
+    values::{to_py_object, to_wasm_value, Value},
+    wasmer_inner::wasmer,
+};
+use pyo3::{
+    exceptions::{RuntimeError, ValueError},
+    prelude::*,
+};
+
+#[pyclass(unsendable)]
+pub struct Global {
+    inner: wasmer::Global,
+}
+
+impl Global {
+    pub fn raw_new(inner: wasmer::Global) -> Self {
+        Self { inner }
+    }
+
+    pub(crate) fn inner(&self) -> &wasmer::Global {
+        &self.inner
+    }
+}
+
+#[pymethods]
+impl Global {
+    #[new]
+    fn new(store: &Store, value: &Value, mutable: Option<bool>) -> Self {
+        let store = store.inner();
+        let value = value.inner().clone();
+
+        Self {
+            inner: match mutable {
+                Some(true) => wasmer::Global::new_mut(store, value),
+                _ => wasmer::Global::new(store, value),
+            },
```

Comment:
You can also do

```Rust
let mutable = mutable.unwrap_or_default();
```

and then have a normal boolean value.  Very small change, but the structure is a bit closer to how default arguments work

Reply:
Correct, it could be easier to read.